.travis.yml

```diff
@@ -29,6 +29,11 @@ jobs:
           - libnotify4
           - libxslt1.1
           - libvpx5
+          # gstreamer and plugins to support video playback in WebKit.
+          - gstreamer1.0-gl
+          - gstreamer1.0-plugins-base
+          - gstreamer1.0-plugins-good
+          - gstreamer1.0-plugins-bad
```

Comment:
where's ugly?..

Reply:
Turns out it's not required! I'd also like to switch to just base and good which is what stock libwebkit2gtk depends on but that requires patching jhbuild and better be done upstream